### ISSUE TITLE
Do you plan to release the checkpoints?

### ISSUE THREAD

**Issue by poonehmousavi** [Hzfinfdu/Diffusion-BERT#23]
Hi,

I try to train a model on a different dataset but the loss doesn't change that much. I wonder if you could release the checkpoints so could first load the model and then finetune it on my own dataset?

Thanks
